.github/workflows/release-python.yml

```diff
@@ -90,10 +89,45 @@ jobs:
         name: wheels
         path: dist
 
+  windows: 
+    runs-on: windows-latest 
+    strategy: 
+      matrix: 
+        python-version: ['3.8', '3.9', '3.10', '3.11'] 
+        target: [x64, x86] 
+    steps: 
+      - uses: actions/checkout@v2 
+      - uses: actions/setup-python@v2 
+        with: 
+          python-version: ${{ matrix.python-version }} 
+          architecture: ${{ matrix.target }} 
+      - name: Update rustup 
+        run: rustup self update 
+      - name: Install Rust toolchain 
+        uses: actions-rs/toolchain@v1 
+        with: 
+          toolchain: stable 
+          target: x86_64-pc-windows-msvc 
+          profile: minimal 
+          default: true 
```

Comment:
This is inconsistent with https://github.com/rigetti/quil-rs/pull/280#pullrequestreview-1614323964, which uses a different action to install a different toolchain.

Maybe it would be worth creating our own [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) to standardize how we do this?

Reply:
To close the loop here, I've reverted the action change in the quil-rs PR, it was only meant to be a temporary change for testing purposes.